Source/ProgressionSystemRuntime/Private/Widgets/PSOverlayWidget.cpp

```diff
@@ -108,5 +117,5 @@ void UPSOverlayWidget::TickPlayFadeOverlayAnimation()
 void UPSOverlayWidget::SetOverlayItemsVisibility(ESlateVisibility VisibilitySlate)
 {
 	// Level is unlocked hide the blocking overlay
-	PSCOverlay->SetVisibility(VisibilitySlate);
+	this->SetVisibility(VisibilitySlate);
```

Comment:
please remove `this->`

Reply:
done with https://github.com/h4rdmol/ProgressionSystem/pull/34/commits/c64bce5118da025fdcfdf728ddc6c97e16cc2b76